### PR TITLE
Setup project with webpack and setting up the prroject virew

### PR DESCRIPTION
**Pull request by fasaldana**
The following tasks have been made:
- Setup basic requirements with webpack
- Created the basic structure of the project with HTML and CSS
- Added basic functionality with JS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-"# todo_list" 
+# Awesome Books
```
Comment (nicupop729) [inline README.md]: The project we work through this week is a simple to-do list app, and not the Awesome Books app. :bookmark: 

- [ ] Please update the name of your app to match the app you work on. :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+
+> This project is as implified todo list crud.
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
```
Comment (nicupop729) [inline README.md:40]: Our Readme should follow the template which Microverse provides for us. We should not omit sections from it. :sun_with_face: 

- [ ] Please introduce the sections missing from your README, and please make sure to not miss the steps needed to replicate your app on another machine. :100: 

Microverse readme template you can find it [here](https://github.com/microverseinc/readme-template).

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @fasaldana, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @fasaldana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**